extensions/amp-pinterest/0.1/save-button.js

```diff
@@ -53,7 +53,8 @@ export class SaveButton {
     this.color = rootElement.getAttribute('data-color');
     this.count = rootElement.getAttribute('data-count');
     this.lang = rootElement.getAttribute('data-lang');
-    this.round = rootElement.getAttribute('data-round');
+    this.round = rootElement.getAttribute('data-round') ||
+        rootElement.getAttribute('height') == rootElement.getAttribute('width');
```

Comment:
Are the `height` and `width` attributes required? If they're not provided by the publisher it'd come down to `undefined === undefined`, is this expected?

Also, not a big deal, but I think the internal JavaScript styleguide asks for strict equality checks when possible :)

Reply:
+1, amp-pinterest supports layouts supported by `isLayoutSizeDefined`, so if the publisher configured `<amp-pinterest layout="fixed-height" ...>` then the `width` attribute could be null

Reply:
Good catch. I added checks to hasAttribute() on both attributes before assessing equality.